src/main/java/com/bp3x/raidbot/commands/RemindCommand.java

```diff
@@ -0,0 +1,65 @@
+package com.bp3x.raidbot.commands;
+
+import com.bp3x.raidbot.commands.util.Event;
+import com.bp3x.raidbot.commands.util.LFGEmbedBuilder;
+import com.bp3x.raidbot.util.RaidBotEmbedBuilder;
+import com.bp3x.raidbot.util.RaidBotRuntimeException;
+import com.jagrosh.jdautilities.command.Command;
+import com.jagrosh.jdautilities.command.CommandEvent;
+import net.dv8tion.jda.api.entities.Member;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Reminder Command to ping accepted players.
+ */
+public class RemindCommand extends Command {
+    private final Logger log = LoggerFactory.getLogger(RemindCommand.class);
+
+
+
+    public RemindCommand() {
+        this.name = "remind";
+        this.help = "Use to remind accepted players";
+        this.arguments = "<eventID> <playerCategory>";
```

Comment:
~~player category might be a bit vague/unclear~~
ah, actually it looks like you don't have the feature to specify which group of players you want to remind. So you can probably just take that out.